06-rmarkdown.Rmd

```diff
@@ -10,18 +10,42 @@ ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_
 
 ## Reports support reproducibility
 
-Using R Markdown files helps you to create supports that can more transparently show what you did for your analysis and it can help you to test that your code works as expected. Scripts allow you to save code, but they do not allow you to have the following additional benefits. 
+Using notebooks help you to create supports that can more transparently show what you did for your analysis and it can help you to test that your code works as expected. Scripts allow you to save code, but they do not allow you to have the following additional benefits.
 
 The following are reasons why R Markdown files help reproducibility:
 
 - They allow you to show and share your code and the output of your code in one place! (this can be done in several ways depending on what you want)
```

Comment:
```suggestion
- They allow you to show and share your code and the output of your code in one place! This can be done in several ways depending on what you want.
```